core/ports/macos/port_interface.cpp

```diff
@@ -332,4 +333,9 @@ namespace Port {
     LOG_ERROR("getIpAddressFromInterfaceName is not implemented");
     return IpAddress();
   }
+
+  void Port::processSocketEvents(HusarnetManager* manager)
```

Comment:
Kinda feels like this should have already been done by the one in `shared_unix_windows`